app/views/lobby/bits.scala

```diff
@@ -121,7 +121,7 @@ object bits:
     nopeInfo(
       h1(trans.sorry()),
       p(trans.weHadToTimeYouOutForAWhile()),
-      p(trans.timeoutExpires(strong(secondsFromNow(ban.remainingSeconds)))),
+      p(trans.timeoutExpires(strong(secondsFromNow(ban.remainingSeconds, "reload")))),
```

Comment:
no need to change `secondsFromNow`.

You can do `secondsFromNow(ban.remainingSeconds)(cls := "extra css classes")`